modules/library/main/src/main/java/org/geotools/filter/expression/PropertyAccessorFactory.java

```diff
@@ -26,6 +26,15 @@
  */
 public interface PropertyAccessorFactory {
 
+    /** The numeric value for highest priority. */
+    int HIGHEST_PRIOIRTY = 0;
+
+    /** The numeric value for lowest priority. */
+    int LOWEST_PRIOIRTY = 5000;
```

Comment:
it seems counter intuitive that the lowest priority has the highest value, you could change that and sort the other way around